net/pkt/Kconfig

```diff
@@ -9,6 +9,7 @@ config NET_PKT
 	bool "Socket packet socket support"
 	default n
 	select NETDEV_IFINDEX
+	select NET_READAHEAD
```

Comment:
@HedongGao shouldn't it be possible to use NET_PKT without NET_READAHEAD?

Reply:
@acassis it's hard to support the realiable recveiving without NET_RADAHEAD. toher similar socket if(udp/tcp/icmp/icmpv6/can) select NET_READAHEAD too.